<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RecordingSettings</class>
 <widget class="QWidget" name="RecordingSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>582</width>
    <height>422</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="recordingFolderGroupBox">
     <property name="title">
      <string>Temporary Folder</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLineEdit" name="recordingFolderLineEdit">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="browseRecordingFolderButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Browse...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="mainGroupBox">
     <property name="title">
      <string>Source</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
      </property>
      <property name="labelAlignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <item row="1" column="0">
       <widget class="QLabel" name="resoulitionLabel">
        <property name="text">
         <string>Resolution</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="resolutionComboBox"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="qualityLabel">
        <property name="text">
         <string>Recording Quality</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="qualityComboBox"/>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="QLabel" name="recordingWarningLabel">
        <property name="text">
         <string>A powerful computer is recommended for &quot;Best&quot; quality and resolution.</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="screenComboBox"/>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="screenLabel">
        <property name="text">
         <string>Screen</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="additionalGroupBox">
     <property name="title">
      <string>Additional Options</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QCheckBox" name="disableAeroCheckBox">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Disable Aero</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="captureCursorCheckBox">
        <property name="text">
         <string>Capture Cursor</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
